serverlessworkflow/modules/ROOT/pages/eventing/working-with-callbacks.adoc

```diff
@@ -87,13 +106,15 @@ The event of type `resume` is consumed by a link:{kogito_sw_examples_url}/server
     }
 ----
 
-This cloud event modifies the original value of the `move` field and publishes it as the `result` field. Moreover, it contains a Cloud Event attribute, named `kogitoprocrefid`, holding the process instance id of the workflow that published the consumed event.
+After that, the published CloudEvent modifies the value of `move` field and publishes the `move` field as a `result` field. Also, the published CloudEvent contains an attribute named `kogitoprocrefid`, which holds the process instance ID of the workflow that published the consumed CloudEvent. 
+
+The `kogitoprocrefid` attribute is crucial because when the correlation is not used, then this attribute is the only way for the Callback state to identify that the related CloudEvent needs to be used to resume the workflow.
 
-As mentioned before, the `kogitoprocrefid` header is pivotal because, when not using correlation, it is the only way for the callback state to know that this event should be used to resume the flow. 
+Note that each Serverless Workflow contains a unique instance ID, which is automatically included in any published CloudEvent. 
```

Comment:
This is not true, kogitoprocrefid has to be INTENTIONALLY added by the external service for the whole thing to work

Reply:
I understood from this sentence that the `kogitoprocinstanceid` is included in every CloudEvent published by the workflow. And it's true, right? The problem here is that the original paragraph referred to the `kogitoprocrefid` that the client response MUST include in their reply to resume the workflow execution.

Reply:
@fjtirado I am not following your comment:

the original sentence was: As mentioned before, the kogitoprocrefid header is pivotal because, when not using correlation, it is the only way for the callback state to know that this event should be used to resume the flow.

Modified sentence: The kogitoprocrefid attribute is crucial because when the correlation is not used, then this attribute is the only way for the Callback state to identify that the related CloudEvent needs to be used to resume the workflow.

I figure that the meaning is same in both sentences.

Reply:
Lets repharse to 

Note that each Serverless Workflow contains a unique instance ID, which is automatically included in any published CloudEvent, as kogitoprocinstance CloudEvent attribute. 

As Ricardo was pointing out the problem of this paragraph is the location.

Reply:
Done